contracts/erc721/PaymentObligation.sol

```diff
@@ -136,15 +138,17 @@ contract PaymentObligation is Initializable, UserMintableERC721 {
     );
 
     // Get the document root from AnchorRepository
-    (bytes32 merkleRoot_, uint32 anchoredBlock_) = super._getDocumentRoot(
+    (bytes32 documentRoot_, uint32 anchoredBlock_) = super._getDocumentRoot(
       anchorId
     );
 
+    bytes32 singatureRoot_ = bytes32(bytesToUint(values[SIGNING_ROOT_IDX]));
```

Comment:
I think it should be called signingRoot instead of signatureRoot
```suggestion
    bytes32 signingRoot = bytes32(bytesToUint(values[SIGNING_ROOT_IDX]));
```